src/main/java/com/google/sps/servlets/EventServlet.java

```diff
@@ -58,4 +83,23 @@ public void doPut(HttpServletRequest request, HttpServletResponse response) thro
     // Redirect back to the HTML page.
     response.sendRedirect("/index.html");
   }
+
+  private String getGCalendarEventsByInterval(String startTime, String endTime) throws IOException {
```

Comment:
This function will called when time is chosen, yes?

Reply:
It's used for getting all users' event from the current view in the interface. So, if the user has the week between 24 and 30 August in the calendar frontend, this function will return all events from that period, events from the user's google calendar.